src/Connectivity/Connectivity.jl

```diff
@@ -294,6 +294,49 @@ function strongly_connected_components end
 end
 
 
+mutable struct ReverPotState{T <: Integer} <: LightGraphs.Traversals.TraversalState
```

Comment:
What does `ReverPotState` short for? We might want to spell it out here.

Also, type parameterization within braces does not use spaces: `{T<:Integer}`. The other spacing is fine.

Reply:
reverse postorder sort

Reply:
Let's name it properly: I suggest `RevPostOrderState`